### DISCUSSION TITLE
QUIC

### DISCUSSION THREAD

**Discussion by Anonym-tsk** [Anonym-tsk/nfqws-keenetic#6] [Идеи и предложения]
Пока нет достоверных подтверждений, что перенаправление/блокировка QUIC на что-то влияет.
Поделитесь опытом, есть ли те, у кого без блокировки UDP/443 не работает, а с ней всё хорошо.
Нужно ли добавлять в скрипты правила для QUIC?

**Comment by dcttly** [comment]
Подтвержу обратное: у меня лыжетелек, ростелеком - все работает и без глушилки QUIC.

**Comment by Anonym-tsk** [comment]
Поддержка QUIC добавлена. Можно переустановить и проверить.

**Reply by Iceled** [reply]
KeeneticOS 4.1.7
Ростелеком
Работает, неправильно настроил изначально.

**Reply by Anonym-tsk** [reply]
> 12 числа работало лучше, на ПК и приставке

А вы уверены, что это как-то связано с обновлением? Можете взять версию от 12 числа и проверить на ней ещё раз

**Comment by novartana** [comment]
Проверил на двух разных Android TV: (TCL C828) и Nvidia Shield. Всё работает идеально.
